### PR TITLE
Suggest a PR for traits rather than issue

### PR DESCRIPTION
**Pull request by StevenClontz**
What do we think about this wording? (See discussion at #1055.)

### CODE REVIEW

**Comment by GeoffreySangston** [comment]
1. What about instead (assuming you know how) to 'Learn how at https://github.com/pi-base/data/wiki/Contributing' ?

> If your proposal is not likely to need much discussion, you should
consider opening a Pull Request to implement this suggestion directly. 
Learn how at https://github.com/pi-base/data/wiki/Contributing.

Or, if available in that format

> ... [on the Wiki](https://github.com/pi-base/data/wiki/Contributing).

2. It would be ideal if there was a brief way to convey which kinds of PR's are unlikely to need much discussion, since people new to pi-base will be reading it. Maybe on the Wiki?

3. Since 'Template' already appears on the screen, we can probably remove the part
> \

(Also, that part for the [Theorem suggestion template](https://github.com/pi-base/data/issues/new?assignees=&labels=theorem&projects=&template=new-theorem-suggestion.md&title=Theorem+Suggestion%3A+%5Bdescription%5D) should be changed to a comment, as it's currently visible italics text.)

Is there a way to inform the reader about which kinds of trait

**Comment by prabau** [comment]
I think it's ok to not show how to do PRs here.  People who know will know how to do it.  People who don't will eventually know how if they care about it.

**Comment by prabau** [comment]
I think the new "NOTE! If your proposal ..." should be at the top and " Modify the below template" should come after, just before what will be modified.

File: .github/ISSUE_TEMPLATE/new-trait-suggestion.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ labels:
 
 <!-- Modify the below template for your trait suggestion: -->
 
+<!-- NOTE! If your proposal is not likely to need much discussion, you should
+consider opening a Pull Request to implement this suggestion directly (assuming
+you know how). -->
+
```
Comment (StevenClontz) [inline .github/ISSUE_TEMPLATE/new-trait-suggestion.md:14]:
```suggestion
<!-- NOTE! If your proposal is not likely to need much discussion, you can
consider opening a Pull Request to implement this suggestion directly, if you feel
comfortable doing so. -->

<!-- Modify the below template for your trait suggestion: -->

```

**Review by GeoffreySangston — APPROVED** [review]
I think it's fine.

**Pull request merged**